.github/workflows/conformance.yml

```diff
@@ -29,6 +29,9 @@ jobs:
   conformance-tests:
     name: Gateway Conformance Tests
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        k8s-version: ["1.23.17", "latest"]
```

Comment:
Can we just do major/minor? Seems easier to maintain and ensures we're always at the latest patch. And in our docs we don't include patch version.